snomed/com.b2international.snowowl.snomed.datastore/META-INF/MANIFEST.MF

```diff
@@ -21,16 +21,14 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.9.0",
  com.b2international.mapdb;bundle-version="[3.0.7,4.0.0)",
  com.b2international.snowowl.snomed.icons;bundle-version="6.15.0",
  com.b2international.snowowl.core;visibility:=reexport,
- com.b2international.snomed.ecl;visibility:=reexport,
- com.b2international.snomed.ql;visibility:=reexport
+ com.b2international.snomed.ecl;visibility:=reexport
```

Comment:
I'm not seeing any version bumps for the ECL 1.5 language (in the root `pom.xml` for example). Am I missing something?